### DISCUSSION TITLE
Will Radio Return Some Day?

### DISCUSSION THREAD

**Discussion by t-aus-m** [meenbeese/Chronos#15] [Ideas]
Hey there,

first of all thanks a lot for bringing this app back to life!
I noticed that you removed the radio feature because it was apparently broken. Now I wonder if there are plans to bring the feature back some time in the future. That would be a huge deal for me because right now there does not seem to be any app that is FOSS and supports alarm clocks with internet radio. 

Thanks again and appreciate your answer :)

**Comment by meenbeese** [comment]
Hello,

I removed the radio feature because, frankly speaking, this is a feature that I have never used/don't care about. Thus, I don't want to spend my time reimplementing it and possibly adding more technical debt to the project. Believe me when I say that there is enough :)

That said, if enough people want it back or someone submits a PR for it, I am open to bringing this feature back. The upvotes and comments on this post can help me decide 😉

**Comment by lespinozaguerrero6-png** [comment]
Anoyher cleam

**Reply by lespinozaguerrero6-png** [reply]
CE-LLI-BACY
